### PR TITLE
cq 2.5.0

### PR DESCRIPTION
**Pull request by lburgazzoli**
- deps: update camel-quarkus to v2.5.0

**Release Note**
```release-note
NONE
```

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -397,7 +397,7 @@
         -->
         <repository>
             <id>camel.quarkus.staging</id>
-            <url>https://repository.apache.org/content/repositories/orgapachecamel-1370</url>
+            <url>https://repository.apache.org/content/repositories/orgapachecamel-1379</url>
```
Comment (squakez) [inline pom.xml:400]: Will conflict with #756

File: pom.xml [hunk 2]
```diff
@@ -435,7 +435,7 @@
         -->
         <pluginRepository>
             <id>camel.quarkus.staging</id>
-            <url>https://repository.apache.org/content/repositories/orgapachecamel-1370</url>
+            <url>https://repository.apache.org/content/repositories/orgapachecamel-1379</url>
```
Comment (squakez) [inline pom.xml:438]: Same here

**Review by squakez — APPROVED** [review]
Good, a part some possible conflict with another PR open

**Review by oscerd — APPROVED** [review]

**Pull request merged**